pkgs/flutter_http_example/ios/Runner/Assets.xcassets/LaunchImage.imageset/README.md

```diff
@@ -0,0 +1,5 @@
+# Launch Screen Assets
+
+You can customize the launch screen with your own desired assets by replacing the image files in this directory.
```

Comment:
Is it typical flutter practice to check in these generated files?

Reply:
It is checked in in the flutter gallery:
https://github.com/flutter/gallery/blob/fa031bfe9d131010e7a56ee5d343f9f85b367d64/ios/Runner/Assets.xcassets/LaunchImage.imageset/README.md?plain=1#L3